flocker/control/schema/types.yml

```diff
@@ -271,3 +271,19 @@ definitions:
       maximum_size:
         '$ref': '#/definitions/maximum_size'
     additionalProperties: false
+
+  dataset_configuration_update:
```

Comment:
This schema allows things the Python method doesn't actually accept, i.e. deleted, dataste_id, metadata.